crd-bootstrap/templates/deployment_crd-bootstrap-controller-manager.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: crd-bootstrap
+    app.kubernetes.io/instance: controller-manager
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: deployment
+    app.kubernetes.io/part-of: crd-bootstrap
+    control-plane: crd-bootstrap-controller-manager
+  name: crd-bootstrap-controller-manager
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: crd-bootstrap-controller-manager
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: manager
+      labels:
+        control-plane: crd-bootstrap-controller-manager
+    spec:
+      containers:
+      - args:
+        - --leader-elect
+        command:
+        - /manager
+        image: ghcr.io/skarlso/crd-bootstrap-controller:latest
```

Comment:
since you already added `image` section to values, the image should not be hardcoded here, but picked up from `values.yaml`